packages/content/src/attributes/types/address/DeliveryBoxAddress.ts

```diff
@@ -64,7 +64,7 @@ export class DeliveryBoxAddress extends AbstractAddress implements IDeliveryBoxA
     public state?: State;
 
     public static override get valueHints(): ValueHints {
-        return ValueHints.from({
+        return super.valueHints.copyWith({
```

Comment:
May I kindly ask you to briefly explain what the purpose of these changes are?

Reply:
Sure.

Here we are "extending" [the valueHints from AbstractAddress](https://github.com/nmshd/runtime/blob/main/packages/content/src/attributes/types/address/AbstractAddress.ts#L22) like we do with the render hints underneath.

This call now adds propertyHints for recipient to the valueHints.

This is just a bugfix and was missed before. With StreetAddress for example [we also do this properly](https://github.com/nmshd/runtime/blob/main/packages/content/src/attributes/types/address/StreetAddress.ts#L66).

Reply:
I you need a deeper explanation I'd suggest to not let this block the PR and we do it afterwards.